tailwind.config.js

```diff
@@ -3,7 +3,43 @@ module.exports = {
   purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
   darkMode: false,
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        gray: {
+          secondary: '#7f8c8d', //paragraphs and prefooter bg
+          light: '#cfcfcf',
+        },
+        white: {
+          DEFAULT: '#FFFFFF',
```

Comment:
I don't think we need to add the white and black DEFAULT as they are already defined in Tailwind.

Reply:
I think because tailwind only has "white" and "black", adding them as objects in the config erases the defaults so we have to add them back. Had an error come up while trying to use the colours without the defaults.